### PR TITLE
Add feature gate for MutatingAdmissionPolicy in v1.30

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enables the MutatingAdmissionPolicy in admission chain.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md:13]: Eeek. Did we also miss documenting an API?

**Review by sftim — COMMENTED** [review]
/sig api-machinery

File: content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md:13]: The gate name shows up in all `kube-xyz --help` output. That means it has been exposed to users, no matter it is fully working or it is not working.

Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md:13]:
```suggestion
---
In Kubernetes {{< skew currentVersion >}}, this feature gate has no effect.
A future release of Kubernetes may use this feature gate name to control behavior.
```

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**